kubernetes-client/src/test/java/io/fabric8/kubernetes/client/behavior/UploadTest.java

```diff
@@ -302,6 +302,7 @@ void bigNumbersSupported(@TempDir Path tempDir) throws Exception {
           // When
           final Path toUploadWithModifiedDate = Files.copy(toUpload, tempDir.resolve("upload-sample.txt"));
           assertTrue(toUploadWithModifiedDate.toFile().setLastModified(9999999999999L)); // Would trigger IllegalArgumentException: last modification time '9999999999' is too big ( > 8589934591 ).
+          assertTrue(toUploadWithModifiedDate.toFile().setLastModified(123456L)); // To have a stable, within ranges, value to compare with
```

Comment:
This is basically rendering the test useless

This test ensures that the changes in https://github.com/fabric8io/kubernetes-client/commit/7a49cce6dd57c69bf9400140b083207d21dd23f2

behave as expected.

It should fail in case the line

```
tar.setBigNumberMode(TarArchiveOutputStream.BIGNUMBER_POSIX);
```

is removed.

We need to find some other way to verify this.

Reply:
Open to suggestions, the test breaks on Java 21 but I don't have much context here ...

Reply:
Do you have any more context about how does the test break in 21?
In any case, it's better to disable this test for Java 21 only (`@DisabledOnJre(JAVA_21)`

Reply:
Agreed, going to disable the test instead, this is the outcome on Java 21:

```
java.lang.AssertionError: 
Expecting
  org.apache.commons.compress.archivers.tar.TarArchiveEntry@df31e1be
to have a property or a field named "lastModifiedTime" with value
  2286-11-20T17:46:39.999Z
but value was:
  2262-04-11T23:47:16.8547758Z
```

Reply:
Not sure if this is actually something misbehaving in that JDK

Reply:
This test has been failing for me locally as well for a while now.